bindings/java/pom.xml

```diff
@@ -52,8 +52,8 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
-        <!-- Use camelCase instead of separating by dots to workaround PowerShell parsing. -->
-        <jniClassifier>${os.detected.classifier}</jniClassifier>
+        <jni.classifier>${os.detected.classifier}</jni.classifier>
```

Comment:
ServerId contains `.` so to support pwsh we must quote something. Let's keep consistency with normal property style.